File: LeetCode/009 Palindrome Number/Solution.java
public class Solution {
    public boolean isPalindrome(int x) {
        if (x<0){return false;}
        int med = x;
        int rslt = 0;
        while(med!=0){
            int tmp = med%10;
            rslt = rslt*10 + tmp;
            med=med/10;
        }
        
        if (rslt == x){
            return true;
        }
        return false;
    }
}

File: LeetCode/009 Palindrome Number/Solution.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False

        copy, reverse = x, 0

        while copy != 0:
            reverse = reverse*10 + copy%10
            copy = copy//10    # 121 = > if single / then results in 12.1,
                               #         but // results in 12

        return x == reverse


if __name__ == "__main__":
    print(Solution().isPalindrome(12))
    print(Solution().isPalindrome(121))
    print(Solution().isPalindrome(13531))
